------------------------------------------------------------
revno: 14069
revision-id: squidadm@squid-cache.org-20160723121351-iuc8hwstrqd0l1dv
parent: squid3@treenet.co.nz-20160723071930-cemledcltg8pkc28
committer: Source Maintenance <squidadm@squid-cache.org>
branch nick: 3.5
timestamp: Sat 2016-07-23 12:13:51 +0000
message:
  SourceFormat Enforcement
------------------------------------------------------------
# Bazaar merge directive format 2 (Bazaar 0.90)
# revision_id: squidadm@squid-cache.org-20160723121351-\
#   iuc8hwstrqd0l1dv
# target_branch: http://bzr.squid-cache.org/bzr/squid3/3.5
# testament_sha1: c9e37a723686ae2ee489ba7ec2e981ae153bda28
# timestamp: 2016-07-23 12:50:56 +0000
# source_branch: http://bzr.squid-cache.org/bzr/squid3/3.5
# base_revision_id: squid3@treenet.co.nz-20160723071930-\
#   cemledcltg8pkc28
# 
# Begin patch
=== modified file 'src/tests/stub_CacheDigest.cc'
--- src/tests/stub_CacheDigest.cc	2016-07-23 07:16:20 +0000
+++ src/tests/stub_CacheDigest.cc	2016-07-23 12:13:51 +0000
@@ -29,3 +29,4 @@
 void cacheDigestGuessStatsReport(const CacheDigestGuessStats *, StoreEntry *, const char *) STUB
 void cacheDigestReport(CacheDigest *, const char *, StoreEntry *) STUB
 uint32_t cacheDigestCalcMaskSize(uint64_t, uint8_t) STUB_RETVAL(1)
+

